src/server/services/openAI.ts

```diff
@@ -0,0 +1,26 @@
+import { Configuration, OpenAIApi } from 'openai'
+
+interface OpenAIData {
+  apiKey: string
+  model: 'gpt-3.5-turbo' | 'gpt-4'
+  role: 'user' | 'assistant' | 'system'
+  content: string
+}
+
+class OpenAIService {
+  async create({ apiKey, model, role, content }: OpenAIData) {
```

Comment:
`apiKey` does not need to be passed every time. Set it as a private attribute of the class and initialize it with `Env.OPEN_AI_KEY`

Reply:
> `apiKey` does not need to be passed every time. Set it as a private attribute of the class and initialize it with `Env.OPEN_AI_KEY`

That is right! I have updated it here https://github.com/kixlab/kuiz/pull/81/commits/5bb867055c576b9f58ca13339587e8a6a4f79c1f :)